sdcm/cluster.py

```diff
@@ -2291,6 +2291,7 @@ def download_monitor_data(self):
             try:
                 self.stop_scylla_monitoring(node)
                 self.download_monitoring_data_dir(node)
+                self.start_scylla_monitoring(node)
```

Comment:
I don't think we should stop it at first place...
Is it requirement to download data?

Reply:
If we don't stop Prometheus before copying, the database files may be copied with missing data or even may include data corruption. 
For example: Prometheus is writing data to the DB files and FS write operation is in progress. The same time you rsync the file. In this scenario you will have partial data in the file in best case (in worst case the data will be corrupted)

Reply:
Theoretically yes, but I don't think it'll happen in our case.
Anyway, I'm ok with starting it.